includes/Modules/Sign_In_With_Google.php

```diff
@@ -238,7 +238,7 @@ private function render_signin_button() {
 			return;
 		}
 
-		$redirect_url = site_url( '/auth/google' );
+		$redirect_url = add_query_arg( 'action', 'google_auth', wp_login_url() );
```

Comment:
This will cause the sign in button tests to fail unless you update the url:

https://github.com/google/site-kit-wp/blob/c7c880fbc25fd49e6f9f17eb8259c3eb507b735c/tests/phpunit/integration/Modules/Sign_In_With_GoogleTest.php#L307